wgpu/src/lib.rs

```diff
@@ -282,6 +284,7 @@ pub struct Surface {
     context: Arc<C>,
     id: ObjectId,
     data: Box<Data>,
+    config: Mutex<Option<SurfaceConfiguration>>,
```

Comment:
Could you add a note about why we need this?

Reply:
Sure. 👍